### PR TITLE
Minimum Sell Amounts

### PR DESCRIPTION
**Pull request by kevin-fruitful**
Implement minimum sell amounts allowed on the marketplace to prevent griefing with really small order amounts. 

Mentioned in Immunify 24703.

### CODE REVIEW

**Comment by kevin-fruitful** [comment]
Task linked: [CU-86937rrp9 `MarketFacet` is not configured with the correct Dust causing the market to be DOSed (IM-24703)](https://app.clickup.com/t/86937rrp9)

File: src/facets/AdminFacet.sol [hunk 1]
```diff
@@ -105,4 +106,8 @@ contract AdminFacet is Modifiers {
     function removeFeeSchedule(bytes32 _entityId, uint256 _feeScheduleType) external assertPrivilege(LibAdmin._getSystemId(), LC.GROUP_SYSTEM_ADMINS) {
         LibFeeRouter._removeFeeSchedule(_entityId, _feeScheduleType);
     }
+
+    function objectMinimumSell(bytes32 _objectId) external view returns (uint256) {
+        return LibAdmin._objectMinimumSell(_objectId);
+    }
```
Comment (amarinkovic) [inline src/facets/AdminFacet.sol]: Why not make this a `getMinimumSell` in the `MarketFacet`, setter is already there?

File: src/libs/LibAdmin.sol [hunk 2]
```diff
@@ -172,4 +176,9 @@ library LibAdmin {
 
         emit FunctionsUnlocked(lockedFunctions);
     }
+
+    function _objectMinimumSell(bytes32 _objectId) internal view returns (uint256) {
+        AppStorage storage s = LibAppStorage.diamondStorage();
+        return s.objectMinimumSell[_objectId];
+    }
```
Comment (amarinkovic) [inline src/libs/LibAdmin.sol]: Maybe it makes more sense to put this into `LibMarket`?

File: src/facets/AdminFacet.sol (hunk 1 above)
Comment (kevin-fruitful) [reply src/facets/AdminFacet.sol]: I was thinking we could make the getters the same name as the variable, and drop the `get` prefix, but for consistency with the rest of the codebase we could add the get prefix. I'll move everything to Market.

**Comment by amarinkovic** [comment]
I think we need a default value, for the case when it's not defined in the mapping.

**Review by amarinkovic — APPROVED** [review]

**Pull request merged**